Jenkinsfile

```diff
@@ -64,7 +64,7 @@ pipeline{
           '''
         runtests('JAVA11_HOME_DIR')
         withSonarQubeEnv('SONAR_Progress') {
-          runSonarScan('JAVA11_HOME_DIR')
+          runSonarScan('JAVA8_HOME_DIR')
```

Comment:
@SameeraPriyathamTadikonda I thought Sonar requires Java 11 and will soon require Java 17?

Reply:
Yes, You are correct. I'm just testing. Closing this PR